discord_bot.py

```diff
@@ -21,37 +20,38 @@ def load_env():
 import argparse
 import json
 import logging
+
 logging.basicConfig(level=logging.DEBUG)
 from pathlib import Path
 
 import discord
 
-from rubber_duck import Message, RubberDuck, MessageHandler, ErrorHandler
+from rubber_duck import Message, RubberDuck, MessageHandler, RubberDuckConfig
 from quest import create_filesystem_manager
 from bot_commands import BotCommands
 
-LOG_FILE = Path('/tmp/duck.log')  # TODO - put a timestamp on this
-
-
-def parse_blocks(text: str, limit=2000):
+LOG_FILE = Path('/tmp/duck.log')  # TODO - put a timestamp on this... on the file
+def parse_blocks(text: str, limit=1990):
     tick = '`'
     block = ""
     current_fence = ""
     for line in text.splitlines():
-        if len(block) + len(line) > limit - 3:
+        if len(block) + len(line) > limit:
```

Comment:
Can we combine the logic of the two `if` statements? Something like:
```python
if len(block) + len(line) > limit or line.strip.startswith(tick*3):
    # close current block and yield it
    # handle whether block is mid-code fence or not

block += ('\n' + line) if block else line
```
Does that simplify the code?